packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md

```diff
@@ -27,149 +27,153 @@ In general, use OpenID Connect to sign users in to apps, and use API Access Mana
+> **Note:** If a user requests scopes from the authorization server that aren't configured, Okta returns an error.
 
-##### Securing applications
+#### Secure applications
 
-* Access tokens and refresh tokens are sensitive and should be protected as such. They should never be stored in client-side or frontend code.
-* A client (application) should never inspect the contents of an access token. For example, do not customize the client's UI based on scopes in the access token. The access token isn't meant for the client to read, it's meant for the client to use. However, there is still a large amount of metadata that may be attached to a token, such as its current validity, approved scopes, and information about the context in which the token was issued. Use the [token introspect endpoint](/docs/reference/api/oidc/#introspect) to access this data.
-* Access tokens should be used exclusively via an HTTP Authorization header instead of encoded into a payload or URL that may be logged or cached.
-* A client secret is a password and should be treated and protected as such. Therefore, it shouldn't be embedded in mobile applications, frontend JavaScript applications, or any other scenario where an attacker could access it.
-* Avoid using the resource owner password grant type (`password`) except in legacy application or transitional scenarios. The authorization code, implicit, or hybrid grant types are recommended for most scenarios.
-* For mobile and single-page web applications, using the authorization code grant type with PKCE is the best practice. The implicit or hybrid grant type is the next best option.
+* Protect access tokens and refresh tokens. They contain sensitive information. Don't ever store them in client-side or front-end code.
+* A client (application) should never inspect the contents of an access token. For example, don't customize the client's UI based on scopes in the access token. The access token isn't meant for the client to read, it's meant for the client to use. However, there's still a large amount of metadata that Okta can attach to a token. For example, Okta may include metadata such as current validity, approved scopes, and information about the context in which Okta issues the token. Use the [token introspect endpoint](/docs/reference/api/oidc/#introspect) to access this data.
+* Use access tokens exclusively through an HTTP Authorization header instead of encoded into a payload or URL that may be logged or cached.
+* A client secret is a password and you should protect it. Therefore, don't embed access tokens in mobile applications, front-end JavaScript applications, or any other scenario where an attacker could access it.
+* Avoid using the Resource Owner Password grant type (`password`) except in legacy application or transitional scenarios. Okta recommends the authorization code, implicit, or hybrid grant types for most scenarios.
```

Comment:
```suggestion
* Avoid using the Resource Owner Password grant type (`password`) except in legacy applications or transitional scenarios. Okta recommends the authorization code, implicit, or hybrid grant types for most scenarios.
```

Reply:
Just a comment, I thought we weren't recommending "implicit" grant types anymore (?)